tests/unit/challenges/test_views.py

```diff
@@ -4287,7 +4286,7 @@ def test_get_challenge_phases_by_challenge_pk(self):
                 "is_submission_public": self.challenge_phase.is_submission_public,
                 "codename": self.challenge_phase.codename,
                 "test_annotation": "http://testserver%s"
-                % (self.challenge_phase.test_annotation.url),
+                                   % (self.challenge_phase.test_annotation.url),
```

Comment:
@GrimmeryGonsa can you revert the indentation changes. Were this done by pre-commit?

Reply:
initially my pull request wasnt getting accepted, so i used pycharm to correct indentation, its probably due to that only.
I will try to revert it...